relay.module.d.ts

```diff
@@ -0,0 +1,38 @@
+export declare const QUERY: Uint8Array;
+export declare const BROADCAST: Uint8Array;
+export declare const BINARY_TYPE: number;
+export declare const TEXT_TYPE: number;
+
+export declare const OP: {
+  ANNOUNCE: number,
+  ANNOUNCE_REPLY: number,
+}
+
+export declare const ArrayEq: (a: array, b: array) => boolean;
+export declare const UInt8UserIdToBase64: (userId: Uint8Array) => string;
+export declare const Base64ToUInt8UserID: (userId: string) => Uint8Array;
+export declare const ArrayToStr: (x: array) => string;
+export declare const StrToArray: (x: string) => array;
+export declare const UserIdHex: (userId: Uint8Array) => string;
+
+export declare class Relay {
+  public ws: WebSocket;
+  public onClientDisconnect: ((userId: Uint8Array) => void) | null;
+  public onSubscription: ((userId: Uint8Array) => Relay) | null;
+  public BinaryMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
+  public TextMessageHandlers: { [key: keyof re.OP]: (sender: Uint8Array, message: Uint8Array | object) => void };
```

Comment:
Shouldn't `message` here be a string?

Reply:
Never mind it looks like the code is converting the string to object before passing, so it should just be a generic `Object` type.  Line above, likewise should always be `Uint8Array` type from what I can see.

Reply:
Are you sure about type on `sender`?